.github/CODE_OF_CONDUCT.md

```diff
@@ -15,7 +15,7 @@ We welcome issues reported by the public, and more specifically the community
 of people using this repository.
 The EGI team is small and cannot support all requests equally.
 While we undertake to do everything in our power to respond to issues
-timely, and to prioritise issues based on reasonable requests from
+timeously, and to prioritise issues based on reasonable requests from
```

Comment:
timeously seems not that common in English

Reply:
So I merged this change from Code of conduct in some others repos we have, it's may be an artefact of some more local English :)
I was feeling timely should more have to be used here liked `in a timely manner`. Will ask english colleagues.

Reply:
Addressed.